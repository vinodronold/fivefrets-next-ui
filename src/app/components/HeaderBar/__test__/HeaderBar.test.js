import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import Headerbar from '../HeaderBar'

describe('components/HeaderBar.js', () => {
    it('should contain one div, one span and text fivefrets', () => {
        const _HeaderBar = shallow(<Headerbar />)
        expect(_HeaderBar.find('div')).toHaveLength(1)
        expect(_HeaderBar.find('span')).toHaveLength(1)
        expect(_HeaderBar.find('span').text()).toBe('fivefrets')
    })
    
    it('should match Headerbar Snapshot', () => {
        const _HeaderBar = renderer.create(<Headerbar />)
        expect(_HeaderBar).toMatchSnapshot()
      })
})