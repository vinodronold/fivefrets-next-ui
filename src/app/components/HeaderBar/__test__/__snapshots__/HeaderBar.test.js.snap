// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/HeaderBar.js should match Headerbar Snapshot 1`] = `
<div
  className="jsx-3186897990"
>
  <span
    className="jsx-3186897990"
  >
    fivefrets
  </span>
</div>
`;
